### PR TITLE
Makefiles: add px4-v2-upload-solo

### PR DESCRIPTION
**Pull request by squilter**
allows code to be built and flashed to a solo with a single command.

### CODE REVIEW

File: mk/px4_targets.mk [hunk 1]
```diff
@@ -125,6 +125,11 @@ px4-cleandep: clean
 	$(v) find $(UAVCAN_DIRECTORY) -type f -name '*.d' | xargs rm -f
 	$(v) find $(SKETCHBOOK)/$(SKETCH) -type f -name '*.d' | xargs rm -f
 
+px4-v2-upload-solo: px4-v2
+	scp $(SKETCH)-v2.px4 root@10.1.1.10:/tmp/
```
Comment (squilter) [inline mk/px4_targets.mk]: As is, this would allow a user to upload ArduPlane or ArduRover to a Solo.  That's probably not good.

**Comment by rmackay9** [comment]
I thought the firmware should be put in the /firmware directory instead of /tmp

**Comment by squilter** [comment]
Option one is putting it in `/firmware` and then rebooting manually.  Option 2 is putting it wherever, and then calling the script to load it from there.  I just carried this over from ardupilot-solo, where they seem to have chosen option 2.

File: mk/px4_targets.mk (hunk 1 above)
Comment (guludo) [inline mk/px4_targets.mk]: I see what you mean. I don't like the hard coded approach though as it becomes necessary to keep track of changes to `SKETCH`. Maybe we could get to a solution that satisfies both of us? :-)

**Comment by rmackay9** [comment]
I was hoping @jschall would add a "reviewed" comment but perhaps they're very busy with the hack-a-thon this week.  I've pushed to master.  thanks!

**Pull request closed without merging**